pkg/server/server.go

```diff
@@ -11,16 +11,18 @@ import (
 )
 
 type KVServerBridge struct {
-	limited *LimitedServer
+	limited             *LimitedServer
+	EmulatedETCDVersion string
```

Comment:
Nit: this isn't used outside this package, make it private
```suggestion
	emulatedETCDVersion string
	limited             *LimitedServer
```